rclpy/rclpy/__init__.py

```diff
@@ -127,10 +127,11 @@ def create_node(
     :param use_global_arguments: ``False`` if the node should ignore process-wide command line
         arguments.
     :param start_parameter_services: ``False`` if the node should not create parameter services.
-    :param initial_parameters: A list of :class:`.Parameter` to be set during node creation.
+    :param parameter_overrides: A list of :class:`.Parameter` which are used to override the
+        initial values of parameters declared on this node.
     :param allow_undeclared_parameters: True if undeclared parameters are allowed, False otherwise.
```

Comment:
```suggestion
     :param allow_undeclared_parameters: True if undeclared parameters are allowed, False otherwise.
        This option doesn't affect `parameter_overrides`.
```